test/torchaudio_unittest/example/__init__.py

```diff
@@ -0,0 +1,8 @@
+import os
+import sys
+
+
+sys.path.append(
```

Comment:
Are there alternatives to this?

Reply:
From the discussion, so as to truly get rid of this, we need to make the test utilities installable package, (it can be a part of torchaudio library). Putting the tests in example directory still requires the similar path modification so that it can access the test utilities defined in `test/torchaudio_unittest`.

The other pros and cons mentioned was that
 - putting example test in `test/torchaudio_unittest`
    - easy to run it in CI
 - putting example tests in `example`
    - better locality